kubetest/main.go

```diff
@@ -657,6 +682,9 @@ func prepare(o *options) error {
 	}); err != nil {
 		return err
 	}
+	if o.parallelism < 0 {
+		o.parallelism = defaultParallelism
+	}
```

Comment:
That's the comment in the flag doc: If `--parallelism` is <1, just set it to `defaultParallelism`. (And also why I warned that it's a little weird.)

(Working on unit test now)